Generator/Sources/NeedleFramework/Parsing/DependencyGraphPrinter.swift

```diff
@@ -0,0 +1,70 @@
+//
```

Comment:
This isn't really part of parsing right? Maybe move this to utility or something?